src/libs/ajax.js

```diff
@@ -1129,7 +1129,10 @@ const Apps = signal => ({
       create: ({ diskName, appType, namespace, bucketName, workspaceName }) => {
         const body = {
           diskConfig: {
-            name: diskName
+            name: diskName,
+            labels: {
+              hidden: 'true'
```

Comment:
'Hidden' is a behavior specific to the UI, but that behavior might change, while these labels become permanent server state, so we should be careful. I think it would be better to store information about what they _are_, then let the code map that to some behavior. E.g., something like `saturnApplication: 'galaxy'` (the 'saturn' prefix is a courtesy since an API user might want to add their own labels)

Reply:
Sounds good, I kept going back and forth on this, so good to know that you prefer this way! Will change it!